packages/keyring-eth-ledger-bridge/src/ledger-keyring.test.ts

```diff
@@ -95,17 +96,17 @@ describe('LedgerKeyring', function () {
    */
   async function basicSetupToUnlockOneAccount(accountIndex = 0): Promise<void> {
     keyring.setAccountToUnlock(accountIndex);
-    await keyring.addAccounts();
+    await keyring.addAccounts(1);
     jest
       .spyOn(keyring, 'unlock')
-      .mockResolvedValue(fakeAccounts[accountIndex] as string);
+      .mockResolvedValue(fakeAccounts[accountIndex] as Hex);
```

Comment:
Very nit: I wonder if we could make `fakeAccounts` a `Hex[]` instead? That would avoid type-casting when ever we use them in the test (we were already type-casting, so I'm ok keeping this as-is too).

Reply:
If we use `fakeAccounts: Hex[]` instead of `fakeAccounts = [..] as const` (readonly constant) then we'll have to non-null assert (`!`) in all places where we access the array with an arbitrary index

Reply:
I was not aware that `as const` would statically enforce the array static content (which makes total sense actually, thanks for sharing this 😄).